### PR TITLE
translation: /conteng/docs/examples/multicluster/index.md

### PR DESCRIPTION
**Pull request by zy475459736**
make a commit per pr

### CODE REVIEW

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 945094af64920cdbe4e020249e2f8697a130298f

https://app.netlify.com/sites/preliminary-istio/deploys/5b76a8293813f033a7ac08bd

File: content_zh/docs/examples/multicluster/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: 多集群功能的赋能
```
Comment (rootsongjc) [inline content_zh/docs/examples/multicluster/index.md]: 标题翻译的不对，应该修改成“启用多集群”

**Comment by rootsongjc** [comment]
CI failed with the following outputs.
```
mdspell 1.3.1
mdl 0.5.0
htmlproofer 3.9.1
    content_zh/docs/examples/multicluster/index.md
        7 | page_icon: /img/examples.svg 
        7 | page_icon: /img/examples.svg 
        7 | page_icon: /img/examples.svg 

>> 3 spelling errors found in 208 files
To learn how to address spelling errors, please see https://github.com/istio/istio.github.io#linting
Running ["ScriptCheck", "LinkCheck", "OpenGraphCheck", "ImageCheck"] on ["./public"] on *.html... 

```

File: content_zh/docs/examples/multicluster/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (geeknoid) [inline content_zh/docs/examples/multicluster/index.md]: This file should be called _index.md

**Comment by zy475459736** [comment]
It seems only when I deleted the line "page_icon: /img/examples.svg" can I pass the ci/cirecleci.
I am wondering why is it.

**Pull request closed without merging**